python/syntax.lark

```diff
@@ -94,12 +97,13 @@ value: NUMBER
      | char
      | tupleval
      | listval
+     | recordval
 
 //constants
 BOOLEAN: ("true" | "false")
 COMMENT: "//" /[^\n]/*
-OR: "||"
-AND: "&&"
+OR: ("||" | "|")
+AND: ("&&" | "&")
```

Comment:
Do we want to make | and || different?

Reply:
in the JS branch they're the same; "and" either becomes bitwise and for ints and doesn't short circuit or becomes logical and for bools and does short circuit

Reply:
ok epic